src-docs/state.py.md

```diff
@@ -122,6 +134,7 @@ The Wazuh server charm state.
 **Attributes:**
  
  - <b>`agent_password`</b>:  the agent password. 
+ - <b>`cluster_key`</b>:  the cluster key. 
```

Comment:
Perhaps describe this a bit more, currently the description seems to restate the name which doesn't add any further information